js/mainScript.js

```diff
@@ -81,5 +75,4 @@ addBtn.addEventListener('click', () => {
 window.onload = () => {
```

Comment:
- [ ] The removeBook function does not work as expected. I removed all the books by clicking on each book button, but there is still a book that is not removed from local storage. Please ensure each book is removed from the local storage after clicking on the remove button.

Reply:
Thank you @medaminedev66  for giving me hints on window.load event.